<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWidget</class>
 <widget class="QWidget" name="ChatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chatWidget</string>
  </property>
  <widget class="QListWidget" name="friendList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>331</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1001</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>photo/1.jpg</normaloff>photo/1.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1002</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>photo/2.jpg</normaloff>photo/2.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1003</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>photo/3.jpg</normaloff>photo/3.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1004</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>photo/4.jpg</normaloff>photo/4.jpg</iconset>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1005</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>photo/5.jpg</normaloff>photo/5.jpg</iconset>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="chatEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
